fastpath/fastpath/core.py

```diff
@@ -1654,7 +1654,14 @@ def process_measurement(msm_tup, buffer_writes=False) -> None:
         assert msmt_uid
         if measurement is None:
             measurement = ujson.loads(msm_jstr)
-        if sorted(measurement.keys()) == ["content", "format"]:
+
+        is_verified = g(measurement, 'is_verified', False)
```

Comment:
It would be great if at some point we renamed this function `g` as something a bit more understandable. It's basically getting the key and if not it's setting to to False. Correct?

I believe modern versions of python have native support for this.

I guess we can take note of it as a future refactoring work.

Reply:
We can just do that I think, the fastpath now runs in python 3.12 in AWS, this can be replaced with `measurement.get('is_verified', False)` 

We can open another issue for that